File: datasets/java/valid/Valid0349.java
public class Valid0349 {
    private int value;
    
    public Valid0349(int value) {
        this.value = value;
    }
    
    public int getValue() {
        return value;
    }
    
    public static void main(String[] args) {
        Valid0349 obj = new Valid0349(42);
        System.out.println("Value: " + obj.getValue());
    }
}

File: datasets/python/valid/valid_0349.py
class Analyzer:
    def __init__(self, temp):
        self._temp = temp

    def get_temp(self):
        return self._temp

    def set_temp(self, acc):
        self._temp = acc

obj = Analyzer(35)
print(obj.get_temp())
obj.set_temp(35)
print(obj.get_temp())
